### ISSUE TITLE
Change Default Indicator color to any other color except black and white 

### ISSUE THREAD

**Issue by mukeshbadgujar** [amcharts/amcharts5#611]
change it for all indicators, (newly added)

default median price color is coming as black put default any other color except black and white if dark mode or day mode. black line not visible in dark mode and white line will not be visible in day mode.

**Comment by martynasma** [comment]
Good catch.

1) Right now indicator colors are hardcoded. Starting with next version (which we'll release this week), we will apply theme-based colors.

This means that if you are using a Dark theme, colors in the "black/white" indicators will invert.

2) You can also customize colors for indicators yourself, using a custom theme:

```JavaScript
class MyTheme extends am5.Theme {
  setupDefaultRules() {

    this.rule("MedianPrice").setAll({
      seriesColor: am5.color(0xff0000)
    });

  }
}

root.setThemes([
  am5themes_Animated.new(root),
  MyTheme.new(root)
]);
```

More about custom themes:
https://www.amcharts.com/docs/v5/concepts/themes/creating-themes/

And here's a default theme for Stock Chart, for reference on defaults, and where to grab code from for overrides:
https://github.com/amcharts/amcharts5/blob/master/src/.internal/charts/stock/StockChartDefaultTheme.ts#L643-L1105

**Comment by martynasma** [comment]
Example:
https://codepen.io/team/amcharts/pen/GRdqXOo??editors=0010

**Comment by martynasma** [comment]
Fixed in [5.2.22](https://github.com/amcharts/amcharts5/releases/tag/5.2.22).

## [5.2.22] - 2022-09-15

### Added
- ZigZag indicator added to `StockChart`.

### Changed
- `"%"` number formatting modifier will now consider locale when choosing whether percent sign goes before or after the number. At this moment only `tr_TR` (Turkish) locale is placing percent sign before number.
- `StockChart` indicators, that were using black (`0x000000`) color as a default series color now use `alternativeBackground` color of a theme (they'll now be white if "Dark" theme is used).

### Fixed
- Stock toolbar dropdown items will now display proper `title` attribute.
- HTML tooltips were flashing an unpositioned content briefly.
- The `"Z"` date formatting option was not taking into account Root element's `timezone` setting.
- If a timezone was set on a chart and data grouping was on, the grouping was not paying attention to the timezone.
- If a timezone which was not using DST was set on a chart, and user was in a timezone which used DST, the axis labels were shifted by 1 hour in some setups.
- `GaplessDateAxis` was not working properly with a timezone set.
- `DateAxis` fills that should have been visible were hidden in some cases.
- `StockChart` was incorrectly switching to percent mode when last of the compared series was removed.
- `"p"` number formatting modifier added. Works the same way as `"%"` except it treats numbers as absolute values and does not multiply them by 100.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
